### ISSUE TITLE
Bug: Import Error in write2us_test.py

### ISSUE THREAD

**Issue by OnurSefa** [bounswe/2021SpringGroup10#100]
There is an import error in the write2us_test.py line 5:
```python
import unittest
import xmlrunner
from HtmlTestRunner import HTMLTestRunner
import os
import main
```
I assume you used the default name 'main' in your feature branch and implemented your unit tests according to. But after merge operation you should obey the discussed directory and file hierarchy. I assume you want to import app.py .
Be careful though, the paths of the apis and your routes may be changed, again, because of the merge operation.

**Comment by tahakalkanli** [comment]
Fixed this bug and opened the pull request #102. Thank you for your attention.

**Comment by OnurSefa** [comment]
I have reviewed your changes and created a review report in #102 which approves your upgrades. Thank you for your effort.
